Website updates

srcarter3 opened the pull request:
Changes for the d2d-perspective and maps-views-projections webpages

tiffanycmeyer13 committed:
Merge pull request #309 from srcarter3/website-updates

Small change to CAVE installation page:

Commit:
Changes to the d2d-perspective webpage
- in the "Hold-Right-Click Resource Name for Menu" subsection, removed the parenthesis text right-clicking to toggle visibility since this is incorrect.  The correct usage is left-clicking to toggle visibility and that appears correctly in a previous subsection.
- After the gif in that same section, added text explaining this menu also has the option for unloading the selected dataset, which might be useful
- In the Display Menu section, moved the order of Sampling Loaded Resources to come before the Two Panel Display subsection
- in the 2-panel section removed the original text in parentheses talking about loading data since I explain in more detail now.
- Added a "Note" about how the cursor does the same in both panels in the 2 panel display
- added a screenshot for this functionality
- added a text explaining data loads to both panels by default but there is an option to load to a specific panel
- added a screenshot for the option to load to a specific panel
- added text pointing out the the ability to pick one panel to load to
- added screenshot of the panel with the L in bottom left
- added new subsection with text and screenshot for unloading data using the Display Menu

Commit:
Merge branch 'unidata_18.1.1' of github.com:Unidata/awips2 into website-updates
merging with latest 18.1.1 changes

Commit:
Small changes to the map-views-projections webpage:
- added a subsection to the Map Editor for Adding a New editor and for Renaming an Editor
- Added text to be a little more clear when referring to the file menu (in the add map section as well as the add projection section)
- replaced the old gif for adding a tab with a new one that reflects the current menu options
- added a new gif for renaming map tabs

Added all the images that were supposed to go in with the previous commit for updates to the d2d perspective page

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Unidata/awips2?pullRequest=312) <br/>All committers have signed the CLA.

Commit:
Small change for d2d perspective page:
- slight change to the description for Unload All Products and Unlaod Graphics for the Display Menu to be more in line with the Raytheon user manual
